filesystem/src/main/java/org/frankframework/filesystem/MailListener.java

```diff
@@ -95,7 +95,7 @@ public Message extractMessage(@Nonnull RawMessageWrapper<M> rawMessage, @Nonnull
 					throw new ListenerException("unable to create XmlWriter", e);
 				}
 
-				try (SaxElementBuilder emailXml = new SaxElementBuilder("email", msgBuilder.asXmlWriter())) {
+				try (SaxDocumentBuilder emailXml = new SaxDocumentBuilder("email", msgBuilder.asXmlWriter(), false)) {
```

Comment:
SaxElementBuilder closed de xmlWriter parameter niet, wat een fout generereert op regel 111. SaxDocumentBuilder werkt hier wel naar behoren

Reply:
Dat is wel verklaarbaar op zich, want het sluiten van een element sluit niet het document en dus niet de stream waar je het document naartoe schrijft.
Bij een sluiten van een document builder is het document compleet en mag de stream dus worden gesloten.